ssg/constants.py

```diff
@@ -61,6 +61,9 @@
 anaconda_system = "urn:redhat:anaconda:pre"
 cce_uri = "https://nvd.nist.gov/cce/index.cfm"
 stig_ns = "https://public.cyber.mil/stigs/srg-stig-tools/"
+cis_ns = "https://www.cisecurity.org/benchmark/red_hat_linux/"
```

Comment:
So... would the CIS references for OCP show this as well? How could we select a different link for different products? We'd need an OpenShift-specific one, or https://www.cisecurity.org/benchmark/kubernetes/ in the meantime since the OCP one is still not on the CIS website.

Reply:
Hmm, great observation, the References URIs are not managed centrally.

>  How could we select a different link for different products?

The products can define their own reference URIs in `<product>/transforms/constants.xslt`.
This is the [rhel8](https://github.com/ComplianceAsCode/content/blob/master/rhel8/transforms/constants.xslt) one, and the [ocp4](https://github.com/ComplianceAsCode/content/blob/master/ocp4/transforms/constants.xslt) is empty.

The `profile_tool.py` script should be improved to be able to get URI specific to the be product it is handling.
I'd leave this for future PRs.

Reply:
"Un-emptying' the CIS URI for OCP4 content in #6663